File: loremaster/backend/src/sql/SQLConnect.js
const mysql = require('mysql');

module.exports = {
    SqlConnect: function(){
        var connection = mysql.createConnection({
            host: process.env.HOST,
            database: process.env.DATABASE,
            user: process.env.USER,
            password: process.env.SQLPASSWORD
        })
        connection.connect(function(error){
            if(error){
                console.log(error);
                throw new Error(error);                
                //connection = null
            }
        })
        return connection;
    }
}

File: loremaster/backend/CharacterCreation/CharacterCreation/SqlConnect.cs
using System;

namespace CharacterCreation
{
    public class SqlConnect
    {
        public string Server { get; set; }  
        public string User { get; set; }  
        public string Database { get; set; } 
        public string Port { get; set; }  
        public string Password { get; set; }  

        public MySqlConnection GetConnection()
        {
            return new MySqlConnection(
                $"server={Server};" +
                $"user={User};" +
                $"database={Database};" +
                $"port={Port};" +
                $"password={Password};"
            );
        }
    }
}